### PR TITLE
Foldseek Tutorial (submitted by the deadline) & MGY441 final project submittions (submitted Dec 4th, 6pm) - requested changes made

### PR DESCRIPTION
**Pull request by izumiando**

### CODE REVIEW

**Comment by izumiando** [comment]
Sorry this is on the same pull request, please let me know if I should submit this in a different way...

File: .DS_Store
Comment (ababaian) [inline .DS_Store:1]: Remove hidden file from repo.

File: tools/.DS_Store
Comment (ababaian) [inline tools/.DS_Store:1]: Ditto remove file.

File: tools/foldseek_tutorial.html
Comment (ababaian) [inline tools/foldseek_tutorial.html:1]: Please don't include an `html` file, the submission for the repo should be entirely in MarkDown.

File: tools/foldseek_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### 1. Download protein sequence FASTA file & upload to Foldseek
+
+- First, download the protein sequence FASTA file from the link provided above. This protein sequence was generated from the whole nucleotide sequence of Obelisk_001739_000001_000001 using the R package Biostrings. The script used for this also provided above for reference and reproducibility. 
+- Next, drag the FASTA file into the input box and click "Predict" for "Structure with ESMFold". This will call the ESMFold protein structure predictor made by Meta to predict the structure of the Obelisk. This step is necessary because there are no structures currently available for this Obelisk. If there was, (for example from a previous AlphaFold run,) the structure from that could be used as a search query instead in Step 2 of this tutorial.
+
+![](./img/Foldseek/predict_structure.png)
```
Comment (ababaian) [inline tools/foldseek_tutorial.md]: For each image, the `./` is not needed you should link to the images as `img/Foldseek/predict_structure.png`. Also the "Figure Title" or Figure Text here is not provided so make sure there is text incase the image does not load, that a description is still available.

File: tools/img/.DS_Store
Comment (ababaian) [inline tools/img/.DS_Store:1]: Remove

File: tools/img/Foldseek/.DS_Store
Comment (ababaian) [inline tools/img/Foldseek/.DS_Store:1]: Remove

File: virus/.DS_Store
Comment (ababaian) [inline virus/.DS_Store:1]: rm

File: virus/Anas_Amicus.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
+# Anas Amicus: Ducking into the "Exemplar" Obelisk
+written by: [Izumi Ando](https://github.com/yourname *optional link*)
```
Comment (ababaian) [inline virus/Anas_Amicus.md]: If you're not linking to your github account, delete the link, else add the link.

File: virus/Anas_Amicus.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+### Anas Amicus
+*{Anas Amicus. "The friend of the duck" or "One who favors the duck". 
+This was inspired by the fact that this Obelisk was initally only detected in 
+runs from BioSamples taken from Anas (genus of dabbling ducks) and not in 
+BioSamples of other birds or animals.}*
+
+[Optional: Tutorial 2 Generative AI Cover-art](img/<virusname>/cover.png)
```
Comment (ababaian) [inline virus/Anas_Amicus.md]: pathname is not valid.

File: virus/Anas_Amicus.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+The associated BioProjects each have 9-11 unique BioSamples and a corresponding run. Of the runs that were not detected for this Obelisk, there were 2 other runs with BioSamples from Anas. The rest were runs from BioSamples taken from Chenonetta (another genus of dabbling ducks), Calidris (a genus of Arctic wading birds)
+
+Non-trivially, a substantial percentage of the reads in each SRR run in the Obelisk cluster are categorized as bacterial (SRR8189300: 6.15%, SRR8189301: 14.36%, SRR7239366: 21.42%). Although the publications associated with these datasets assign the sample origin (birds) as the host for all detected viruses, I hypothesize that the host for this Obelisk could potentially be bacteria. An attempt was made to quantitatively assess this hypothesis as discussed in the following section (Question 3, Figures TODO).
+
+### Virus Genome
+
+![**Figure 1. Genomic Characteristics of Anas Amicus**](./img/Anas_Amicus/SRR8189300_genome_vis.png)
```
Comment (ababaian) [inline virus/Anas_Amicus.md]: As with tool, fix path here and throughout document.

**Review by ababaian — CHANGES_REQUESTED** [review]
Please see review comments to "close" the PR and complete the task.

**Comment by izumiando** [comment]
requested changes made

**Review by ababaian — APPROVED** [review]

**Pull request merged**